### ISSUE TITLE
Easy shortcut button for pi desktop [FEATURE]

### ISSUE THREAD

**Issue by undeadmky** [vicwomg/pikaraoke#324]
Easier way to open the program when not set to auto boot. A simple desktop icon that doesn't request how to open file.

**Comment by vicwomg** [comment]
Yeah, this is not super friendly for non-linux fluent folks. But here's how you do it manually:

open a terminal:

```
cd ~/Desktop
touch ./launchPikaraoke.desktop
nano ./launchPikaraoke.desktop
```

Opens nano text editor. Enter the following in the file
```
[Desktop Entry]
Name="Pikaraoke"
Exec="/home/pi/pikaraoke/pikaraoke.sh"
Type=Application
```

ctrl +x to exit and save

**Comment by sezaaru** [comment]
Hi Vic. Is this also applicable to Ubuntu or Debian.

is there an autoboot also for non-raspberry pi devices.

**Comment by safepay** [comment]
Resolved in #714
